facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -243,6 +259,13 @@ private static void handleLaunchArguments(String[] args) {
                 soundEnabled = false;
             } else if (arg.equals(LOAD_LAST_GAME)) {
                 loadLastGame = true;
+            } else if(arg.startsWith(SERVER_PORT)) {
+                try {
+                    serverPort = Integer.parseInt(arg.substring(SERVER_PORT.length()));
+                } catch (NumberFormatException e) {
+                    new TransientConfig().setServerPort(TerasologyConstants.DEFAULT_PORT);
+                    System.out.println("Couldn't parse server port. Using default port.");
+                }
```

Comment:
Again, I am far from being an expert in networking, but if I made a mistake in writing the port number, enough to make it unparseable, I think Terasology should not start and report the error instead of smoothly switching to the default port.

Also, depending on what follows the current warning printout and at what speed, it could be swamped by subsequent startup and log statments, to the point that the user might not realize server port is not what he or she expects.